qa/tasks/cephfs/test_failover.py

```diff
@@ -340,6 +340,50 @@ def test_thrash(self):
 
         self.fs.wait_for_daemons(timeout=90)
 
+class TestFailoverBeaconHealth(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def initiate_journal_replay(self, num_files=100):
+        """ Initiate journal replay by creating files and restarting mds server."""
+
+        self.config_set("mds", "mds_delay_journal_replay_for_testing", "5000")
+        original_active = self.fs.get_active_names()[0]
+        self.mounts[0].test_files = [str(x) for x in range(num_files)]
+        self.mounts[0].create_files()
+        self.fs.fail()
+        self.fs.set_joinable()
+
+    def test_replay_beacon_estimated_time(self):
```

Comment:
There is a (probably unrelated) test failure here: https://pulpito.ceph.com/vshankar-2024-01-10_15:00:23-fs-wip-vshankar-testing-20240103.072409-1-testing-default-smithi/7511461/

Which hints at a kernel umount hang. I've requested @lxbsz to have a look. However, the subsequent tests didn't run since the test case aborted on the failed test, so I'll rerun specific test once @lxbsz updates. Rest of the fs suite tests look good.

Reply:
Looks like the umount hang is due to the MDS still in replay since the test case modified a config to slow down replay. The config needs to be reset and the test should wait for the MDS to be back active before ending (client unmounting).